Makefile

```diff
@@ -66,20 +67,10 @@ dev: DEBUG = desktop:*
 dev: 
 	$(MAKE) start NODE_ENV=$(NODE_ENV) DEBUG=$(DEBUG)
 
-
-BASE_CONFIG := $(THIS_DIR)/desktop-config/config-base.json
-ENV_CONFIG := $(THIS_DIR)/desktop-config/config-$(CONFIG_ENV).json
-
-.PHONY: desktop$/config.json
-desktop$/config.json:
-ifeq (,$(wildcard $(ENV_CONFIG)))
-	$(warning Config file for environment "$(CONFIG_ENV)" does not exist. Ignoring environment.)
-else
-	$(eval EXTENDED = true)
-endif
-	@node -e "const base = require('$(BASE_CONFIG)'); let env; try { env = require('$(ENV_CONFIG)'); } catch(err) {} console.log( JSON.stringify( Object.assign( base, env ), null, 2 ) )" > $@
+desktop$/merged-config-%.json: desktop-config$/config-base.json desktop-config/config-%.json
+	@node -e "const base = require('$(THIS_DIR)/$<'); let env; try { env = require('$(THIS_DIR)/$(word 2,$^)'); } catch(err) {} console.log( JSON.stringify( Object.assign( base, env ), null, 2 ) )" > $@
```

Comment:
`$<` is the first dependency (`desktop-config$/config-base.json`), `(word 2,$^)` is the second (`desktop-config/config-%.json`). This reads a bit Greek -- maybe we can alias them inside the rule (though at first try, I wasn't able to).